scanomatic/models/compile_project_model.py

```diff
@@ -85,11 +84,9 @@ class CompileImageAnalysisModelFields(Enum):
 class CompileImageAnalysisModel(Model):
     def __init__(
         self,
-        image: Optional[CompileImageModel] = None,
-        fixture: Optional[FixtureModel] = None,
+        image: CompileImageModel,
```

Comment:
```suggestion
        *,
        image: CompileImageModel,
```
Models should only accept keyword arguments. But when I re-worked this, I didn't think of that they could be keyword arguments without defaults so if this works then perhaps you could update the other models to with the asterisk.

Reply:
Some of the models caused errors in the factory when forcing keyword arguments, so I did this change only to the models that have falsely Optional parameters

Reply:
and this one caused issues?

Reply:
Oops, no this one was ok, fixed